repos/spack_repo/builtin/packages/gaudi/package.py

```diff
@@ -160,8 +160,11 @@ class Gaudi(CMakePackage, CudaPackage):
     depends_on("cmake@3.19:", type="build", when="@39:40.0")
     depends_on("cmake@3.29:", type="build", when="@40.1:")
     depends_on("cppgsl")
+    depends_on("fmt")
     depends_on("fmt@:8", when="@:36.9")
-    depends_on("fmt@:10")
+    depends_on("fmt@:10", when="@:38")
+    depends_on("fmt@:11", when="@:40")
```

Comment:
```suggestion
```
This is not correct, I can build Gaudi v40.1 and master with fmt 12.1.0 (without tests).

Reply:
As of when does gaudi support fmt-12? Already in v39? Or should this be something like:
```suggestion
    depends_on("fmt@:11", when="@:39")
```

Reply:
It seems 39.4 does not compile with fmt 12. So I think this suggestion above is good.